### ISSUE TITLE
Schedule 4, 5, 8, 10 - Selected row turns blue - row text and buttons should darken - accessibility/visibility issue

### ISSUE THREAD

**Issue by amee99073** [bcgov/nr-ilcr#437]
### Bug Summary

Schedule 4, 5, 8, 10 - Selected row turns blue - row text and buttons should darken 

### Describe the Bug

Schedule 4, 5, 8, 10 - Selected row turns blue - row text and buttons should darken (in dark and light mode)
This is an accessibility/visibility issue. Should behave the same way as when highliting a row via hover

### Screenshots

```shell

```

### Affected Environments

- [ ] Development
- [ ] Staging
- [ ] Production
